test/hooks/achievement-rule-checker.test.js

```diff
@@ -105,6 +105,23 @@ describe('\'achievement-rule-checker\' hook', () => {
     assert.deepEqual(result[0].amount, 1);
   });
 
+  it('gives AnyOf Conditions Achievement', async () => {
+    await app.service('events').create({
+      'name': 'ParameterEvent',
+      'user_id': user_id,
+      'payload': {'x': 1, 'y': 3}
+    });
+
+    const result = await app.service('achievements').find({
+      query: {
+        user_id: user_id,
+        name: 'AnyOfConditionsAchievement'
+      }
+    });
+
+    assert.deepEqual(result[0].amount, 1);
```

Comment:
IMHO at some point it might make sense to introduce a helper method assertAchievement(user_id, achievementName, count) to avoid the code duplication. But I'd rather like to see it in a separate PR.

Reply:
Did this in #55.